core/commands/show_file_at_commit.py

```diff
@@ -92,7 +92,12 @@ def run(self, edit, position=None):
         file_path = settings.get("git_savvy.file_path")
         commit_hash = settings.get("git_savvy.show_file_at_commit_view.commit")
 
-        text = self.get_file_content_at_commit(file_path, commit_hash)
+        if commit_hash and self.savvy_settings.get("blame_follow_rename"):
+            filename_at_commit = self.filename_at_commit(file_path, commit_hash)
+        else:
+            filename_at_commit = file_path
+
+        text = self.get_file_content_at_commit(filename_at_commit, commit_hash)
         render(view, text, position)
```

Comment:
I don't think the changes for "show_file_at_commit" do work.  Not sure I do get them though.  What confuses me is that we get the `filename_at_commit` here but don't use it on L103 and L105 too.  Is that intentional?

Also the whole `show_file_at_commit` view has `[p]revious` and `[n]ext` handlers, starting at L201 below.  Do they just work or did you miss them?

Reply:
I realize there's an error in the code I wrote. It should be checking `self.savvy_settings.get("log_follow_rename"):`, not `self.savvy_settings.get("blame_follow_rename"):`

The code works for me in both cases, but that's because I have both of these enabled in my GitSavvy settings

```json
	"log_follow_rename": true,
	"blame_follow_rename": true,
```

I just fixed this

---

> What confuses me is that we get the filename_at_commit here but don't use it on L103 and L105 too

Thanks for catching this, I just fixed this as well

> Also the whole show_file_at_commit view has [p]revious and [n]ext handlers, starting at L201 below. Do they just work or did you miss them?

I tried making this code work and couldn't